meilisearch-http/src/routes/document.rs

```diff
@@ -29,6 +30,19 @@ macro_rules! guard_content_type {
 }
 
 guard_content_type!(guard_json, "application/json");
+*/
+
+fn guard_json(head: &actix_web::dev::RequestHead) -> bool {
+            if let Some(content_type) = head.headers.get("Content-Type") {
+                content_type
+                    .to_str()
+                    .map(|v| v.contains("application/json"))
+                    // if no content-type is specified we still accept the data as json!
+                    .unwrap_or(true)
+            } else {
+                false
```

Comment:
This is where you want to return true: there is no content type specified

Reply:
Oh yes, you're right!
Strangely, it worked the other way around :grimacing: 
I added a patch. Thank you!